apartmenttherapy/order-service
# Adding the ability to parse tokens from multiple auth0 clients.

#### What does this PR do & why?
Updating the authentication routine to allow multiple auth0 clients.

#### How can this be manually tested?
- Tests should run.
- Should be able to use the order service from the iphone & android apps.

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1112706920978686

#### GIF


#### Have you remembered to...?
